frontend/shared/ExpandableArea.test.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+import ExpandableArea from './ExpandableArea';
+
+describe('<ExpandableArea/>', () => {
+  it('renders', () => {
```

Comment:
Can we add a test if it `isExpanded`?

Reply:
the expansion is still controlled by USWDS JS so, not easily.

Reply:
Understood. Thanks!